framework/src/modules/chain/defaults/genesis_block_devnet.json

```diff
@@ -0,0 +1,1760 @@
+{
```

Comment:
Shouldn't we have a different genesis block been part of framework? Or we use devnet genesis block which comes from Lisk Core our network implementation.

Reply:
It is a Genesis Block for development purposes, so devnet genesis block is correct to me.